job/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -449,6 +450,11 @@ private void finish(@Nonnull Result r, @CheckForNull Throwable t) {
             }
         }
         FlowExecutionList.get().unregister(new Owner(this));
+        try {
+            StashManager.maybeClearAll(this);
+        } catch (IOException x) {
+            LOGGER.log(Level.WARNING, null, x);
```

Comment:
Well, I think this falls into the category of internal errors that get sent to the Jenkins log.